src/fe-text/irssi.c

```diff
@@ -74,11 +75,20 @@ void mainwindow_activity_deinit(void);
 void mainwindows_layout_init(void);
 void mainwindows_layout_deinit(void);
 
+#ifdef HAVE_SECCOMP
```

Comment:
I would get rid of the ifdef here, and instead have it inside the function bodies in `sandbox.c`, like this:
```void drop_privileges(void)
{
#ifdef HAVE_SECCOMP
...
```

Does that makes sense?